src/storage/src/hummock/local_version.rs

```diff
@@ -170,26 +170,26 @@ impl LocalVersion {
 
     pub fn add_sync_state(
         &mut self,
-        sync_epoch: Vec<HummockEpoch>,
+        sync_epochs: Vec<HummockEpoch>,
         sync_uncommitted_data: SyncUncommittedData,
```

Comment:
`sync_epoch` and `sync_uncommitted_data` shoud order their items in descending order of epochs. Let's add a debug assert here.